### PR TITLE
Create AutoSort.yml

### PR DESCRIPTION
**Pull request by ming-tsai**

### CODE REVIEW

File: _data/projects/AutoSort.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: AutoSort
+desc: Sort by a string or by default attributeA brief description of the project
+site: https://github.com/ming-tsai/NetCore.AutoSort
+tags:
+  - c#
+  - netcore
+  - dotnet3-1
```
Comment (shiftkey) [inline _data/projects/AutoSort.yml]: Let's avoid focusing on specific versions of frameworks wherever possible:

```suggestion
  - netcore
```

File: _data/projects/AutoSort.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: AutoSort
+desc: Sort by a string or by default attributeA brief description of the project
```
Comment (shiftkey) [inline _data/projects/AutoSort.yml]: Looks like a copy-paste typo here

```suggestion
desc: Sort by a string or by default attribute
```

File: _data/projects/AutoSort.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+site: https://github.com/ming-tsai/NetCore.AutoSort
+tags:
+  - c#
+  - netcore
+  - dotnet3-1
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/ming-tsai/NetCore.AutoSort/labels/up-for-grabs
```
Comment (shiftkey) [inline _data/projects/AutoSort.yml:9]: Looks like there are no open issues currently for new contributors. Is that something you're able to setup before we merge this?

**Review by shiftkey — COMMENTED** [review]
Thanks for the contribution @ming-tsai! 

A couple of small typos and a question to go before we merge this!

File: _data/projects/AutoSort.yml (hunk 3 above)
Comment (ming-tsai) [reply _data/projects/AutoSort.yml:9]: I have already close it, I will reopen it

**Comment by ming-tsai** [comment]
> Thanks for the contribution @ming-tsai! 
> 
> A couple of small typos and a question to go before we merge this!

Thanks for your comments @shiftkey, I have applied.

**Review by shiftkey — APPROVED** [review]

**Pull request merged**